scripts/start.py

```diff
@@ -166,8 +166,8 @@ def main(args=None):
         stack.enter_context(managed_dev_appserver)
 
         # Wait for the servers to come up.
-        common.wait_for_port_to_be_open(PORT_NUMBER_FOR_GAE_SERVER)
```

Comment:
@U8NWXD, Do you think these changes are okay? (Asking this as you have recently added these functions.)

Reply:
In general, yes. I left a comment on the one bit I disagree with, but the overall PR is correct. SAEb-ai and I have been discussing this together in #11868